src/common-imports.ts

```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue'
```

Comment:
@object-required I think Safydy already said somewhere but I can't find the comment now.. Could you make sure to use this in https://github.com/FLOIP/flow-builder/blob/master/stories/story-utils/storeSetup.ts also. Thanks!